pkgs/development/libraries/imgui/cimgui.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, lib, callPackage, fetchFromGitHub, luajit }:
+
+stdenv.mkDerivation rec {
+  pname = "cimgui";
+  version = "1.87";
```

Comment:
The [never](https://github.com/cimgui/cimgui/releases/tag/1.88) version is out.

Reply:
I switched this to use `imgui.version` so that the two packages move in lockstep.